docs/development.md

```diff
@@ -67,93 +106,147 @@ docker compose down -v
+cd web
 yarn serve
 ```
 
-View main application at `http://127.0.0.1:8081`
+View the main application at `http://127.0.0.1:8081/`. Changes to files in the `web` directory will automatically trigger a reload in your browser. 
+
+> **Note**: the frontend application will still be served via Docker Compose on port `8080`, but it will not pick up changes to the `web` directory automatically. Be aware of which port you are accessing when doing frontend development.
 
-## Why not `localhost`?
+### Why not `localhost`?
 
 It is recommended to use `127.0.0.1` instead of `localhost` for local development. This is because `localhost` is **not** allowed as a redirect URI for an ORCID client. The workaround is to register `127.0.0.1` as a redirect URI with ORCID and to use subsequently visit `127.0.0.1` for local testing.
 
-# Testing
+## Running ingest
+
+> **Note**: This is not generally required unless you are specifically working on the ingest code. If you are working on the web application, simply [loading from a recent production backup](#load-production-data) is sufficient.
+
+1. Ensure that you have completed the sections above about configuring your [NERSC credentials](#nersc-credentials) and [MongoDB credentials](#mongodb-credentials). 
+2. Obtain a new SSH key from NERSC if you haven't done so in the last 24 hours.
+    ```bash
+    sshproxy.sh -u $NERSC_USER
+    ```
+3. Set up an active SSH tunnel to the dev and production MongoDB instances if you do not already have one established.
+    ```bash
+    ssh \
+      -L 37018:mongo-loadbalancer.nmdc-dev.production.svc.spin.nersc.org:27017 \
+      -L 37019:mongo-loadbalancer.nmdc.production.svc.spin.nersc.org:27017 \
+      -o ServerAliveInterval=60 \
+      -f \
+      -N \
+      -l $NERSC_USER \
+      -i ~/.ssh/nersc \
+      dtn01.nersc.gov
+    ```
+    > That command will set up SSH port forwarding such that your computer can access the dev MongoDB server at `localhost:37018` and the prod MongoDB server at `localhost:37019`. 
+  
+    > From within a Docker container `host.docker.internal` can be used to access the `localhost` of your computer. When ingesting from the dev or prod MongoDB instances, be sure to set `NMDC_MONGO_HOST=host.docker.internal` in your `.env` file.
+
+    > See https://github.com/microbiomedata/infra-admin/blob/main/mongodb/connection-guide.md (internal) for more information on connecting to the MongoDB instances.
+4. Create a local copy of ingest support files:
+    ```bash
+    mkdir -p data && scp \
+      -r \
+      -i ~/.ssh/nersc \
+      "$NERSC_USER@dtn01.nersc.gov:/global/cfs/cdirs/m3408/ingest" \
+      data
+    ```
+5. Run the ingest command:
+    ```bash
+    docker-compose run backend nmdc-server ingest -vv --function-limit 100
+    ```
+   
+    > **Note**: The `--function-limit` flag is optional. It is used to reduce the time that the ingest takes by limiting the number of certain types of objects loaded. This can be useful for testing purposes.
```

Comment:
You can skip the gene function ingest altogether with the `--skip-annotation` flag. The only thing this affects for the development environment is the gene function search. I haven't actually run an ingest from a remote mongo database in a while, so I don't know what the time delta is between skipping annotation ingest entirely, limiting to something like 100, or full ingest. I'm wondering if anyone does have an idea of recent time spent on local ingest

Reply:
I have run an ingest from a remote MongoDB to my local Postgres within the last few weeks (a month maybe?), but I don't have exact numbers on how much time is spent one way or the other.

Reply:
Might be worth calling out or at least mentioning that we have useful help text that can be found with `nmdc-server ingest --help`